packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -122,7 +122,7 @@ const AccountSwitcher = ({
         closeAccountsDialog();
         if (account_loginid === loginid) return;
         await switchAccount(loginid);
-        RudderStack.setAccountType(loginid.substring(0, 2));
+        Analytics.setAccountType(loginid.substring(0, 2));
```

Comment:
<img width="1020" alt="image" src="https://github.com/binary-com/deriv-app/assets/129206554/ffafc548-6fd6-4a74-97ce-04c90b86c317">
`setAttributes` is specifically for growthbook, but I introduced this method in the rudderstack instance, so it will work after latest update

check the following
<img width="497" alt="image" src="https://github.com/binary-com/deriv-app/assets/129206554/c8b63c80-9e84-4a4e-a1e0-5145420d97f5">